server/templates/details.html.tmpl

```diff
@@ -56,7 +56,7 @@
 <li data-status="{{$s}}">
   <div class="bg-white p-2 shadow-sm max-w-lg status-stripe {{$s}}">
     {{template "result-details" .}}
-    {{if (or .PredicateResults (hasRequires .Requires))}}
+    {{if (or (.PredicateResults) (or (hasRequires .Requires) (hasRequiresPermissions .Requires)))}}
```

Comment:
In templates `or` can take multiple arguments, so I think this will work:

```suggestion
    {{if (or (.PredicateResults) (hasRequires .Requires) (hasRequiresPermissions .Requires))}}
```

Reply:
Fixed!